docs/sources/mimir/configure/configure-blocked-queries.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Configure queries to block
+description: Control what queries are sent to your Mimir installation.
+weight: 100
+---
+
+# Configure queries to block
+
+In certain situations, you might want to control what queries are being sent to your Mimir installation. These queries
+might be intentionally or unintentionally expensive to run, and they might affect the overall stability or cost of running
```

Comment:
Hi @wilfriedroset, perhaps for my personal learning, but why would a query ever be intentionally expensive to run? It sounds like there is something underneath this idea and the result is that a given query is expensive to run.

Reply:
From a service provider, we might be exposed to denial of service, this is where such features are useful.
For example, a single user looking to cause harm to your service could storm the Mimir cluster with queries that would not be stopped by the current limits.
In that event, either the cluster is scaled or the quality of service decrease.

Most of this work is inspired and/or copied from Loki.
Loki's documentation contains the same phrase: https://grafana.com/docs/loki/latest/operations/blocking-queries/